include/can_interface.h

```diff
@@ -476,6 +631,320 @@ class CANTXMessage : public ICANTXMessage
     }
 };
 
+template <size_t num_groups, size_t num_multiplexors_to_transmit, typename MultiplexorType>
+class MultiplexedCANTXMessage : public ICANTXMessage
+{
+public:
+    template <typename... Ts>
+    /**
+     * @brief Construct a new CANTXMessage object
```

Comment:
Can you add more description in the brief? Since all the CAN message classes have the exact same brief, probably should specify how they're different so people know when to use what.

Reply:
I added some more descriptions